sns-topic-publisher/cfn-templates/sns-topic-publisher.json

```diff
@@ -0,0 +1,175 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Parameters": {
+    "SNSTopicName": {
+      "Type": "String",
+      "Description": "Please enter your SNS Topic Name"
```

Comment:
Add a note to the description that SNS topic must be in the same region as the template